adding new 11ty starters

adamdjbrett opened the pull request:
I added some new starters for 11ty and updated 2 of my previous contributions.

Snapstromegon commented:
Hey, seems like you added the author field to some of the files, but not all - was this intentional?
(and if you add it to all, can you make sure that formatting is consistent?)

adamdjbrett commented:
Dear @Snapstromegon thank you for your comment. I have added author to all of them. Before I close this and push the updated and fixed PR.
Here is what I have for format for all of them does this work?
```
{
	"url": "https://github.com/adamdjbrett/11ty-wisp",
	"name": "Wisp",
	"description": "A minimalist blog template using Pico CSS.",
	"author": "adamdjbrett",
	"demo": "https://wisp.000000076.xyz/"
}
```

Snapstromegon commented:
Hey, no need to create a new PR, you can just update this one (by pushing a new commit to your branch fpr this PR).
I think most starters use 2 spaces indentation, but 4 is fine too, just make it consistent within a file.

adamdjbrett commented:
thank you. I have pushed the changes.
